epam-assets/theme/theme_loveship.scss

```diff
@@ -317,230 +320,17 @@
-    --uui-search-input-bg-disabled: var(--uui-control-bg-disabled);
-    --uui-search-input-border: var(--uui-control-border);
-    --uui-search-input-border-disabled: var(--uui-control-border-disabled);
-    --uui-search-input-border-hover: var(--uui-control_textinput-border-hover);
-    --uui-search-input-border-active: var(--uui-control-border-checked);
-    --uui-search-input-border-error: var(--uui-negative-50);
-    --uui-search-input-placeholder: var(--uui-control-placeholder);
-    --uui-search-input-text: var(--uui-control-text);
-    --uui-search-input-text-disabled: var(--uui-control-text-disabled);
-    --uui-search-input-icon-clickable: var(--uui-icon);
-    --uui-search-input-icon-clickable-hover: var(--uui-icon-hover);
-    --uui-search-input-icon-disabled: var(--uui-icon-disabled);
-    --uui-search-input-border-radius: var(--uui-border-radius);
-
-    /* Popover */
-    --uui-popover-shadow: var(--uui-shadow-level-2);
-
-    /* Footer */
-    --uui-footer-bg: var(--uui-surface-contrast);
-
-    /* FilterPanelItemToggler vars */
-    --uui-filters-toggler-border: var(--uui-text-input-border);
-    --uui-filters-toggler-border-hover: var(--uui-text-input-border-hover);
-    --uui-filters-toggler-border-radius: var(--uui-border-radius);
-    --uui-filters-toggler-bg: var(--uui-surface);
-    --uui-filters-toggler-bg-hover: var(--uui-surface-lower);
-    --uui-filters-toggler-text: var(--uui-text-brand);
-    --uui-filters-toggler-label: var(--uui-text-tertiary);
-    --uui-filters-toggler-icon: var(--uui-icon);
-    --uui-filters-toggler-shadow: none;
-
-    --uui-filters-toggler-font: var(--font-regular);
-    --uui-filters-panel-header-font: var(--font-regular);
-
-    /* Spinner */
-    --uui-spinner: var(--uui-primary-50);
-
-    /* Blocker */
-    --uui-blocker-bg: var(--uui-app-bg);
-
-    /* AvatarStack */
-
-    --uui-avatar-stack-font: var(--font-semibold);
-
-    /* Pagination */
-
-    --uui-pagination-font: var(--font-regular);
-
-    /* FileCard */
-    --uui-file-card-background: var(--night400);
-    --uui-file-card-background-loaading: var(--night400);
-    --uui-file-card-error: var(--uui-negative-50);
-    --uui-file-card-error-block: var(--uui-negative-70);
-    --uui-file-card-default: var(--uui-secondary-50);
-    --uui-file-card-dropstart-border: var(--uui-primary-50);
-    --uui-file-card-dropstart-background: var(--uui-primary-10);
-    --uui-file-card-link-hover: var(--uui-primary-60);
-
-    /* IndeterminateBar */
-
-    --uui-indeterminate-bar-wrapper: var(--uui-secondary-20);
-    --uui-indeterminate-bar-background: var(--uui-primary-50);
-
-    /* ProgressBar */
-
-    --uui-progress-bar-wrapper: var(--uui-secondary-20);
-    --uui-progress-bar-background: var(--uui-primary-50);
-    --uui-progress-bar-label: var(--uui-surface-contrast-lower);
+    /* override variables */
 
-    /* ------------------------------------ */
+    --uui-accordion-border-radius: 6px;
```

Comment:
Need to discuss if we can make a default for this variable and move it on component level. Probably we can use here  --uui-border-radius as a default, but replace it in skins with '6px' value

Reply:
![image](https://github.com/epam/UUI/assets/97472156/b851f60d-93ea-4e28-872a-9ac6402adf11)
We have default value, in this case we override value for theme